code/modules/aberrants/organs/mods/component/_component.dm

```diff
@@ -149,27 +165,36 @@
 
 /datum/component/modification/organ/uninstall(obj/item/I, mob/living/user)
 	..()
+	var/obj/item/organ/O = I
 	if(istype(I, /obj/item/organ/internal/scaffold))
 		var/obj/item/organ/internal/scaffold/S = I
 		S.try_ruin()
 
+	// If the organ has no owner or is still modded, do nothing
+	if(!O.owner || I.item_upgrades.len > 0)
+		return
+
+	var/mob/living/carbon/human/H = O.owner
+	H.mutation_index -= 1
```

Comment:
```suggestion
/datum/component/modification/organ/uninstall(obj/item/organ/O, mob/living/user)
	..()
	if(istype(I, /obj/item/organ/internal/scaffold))
		var/obj/item/organ/internal/scaffold/S = O
		S.try_ruin()

	// If the organ has no owner or is still modded, do nothing
	if(!O.owner || O.item_upgrades.len > 0)
		return

	var/mob/living/carbon/human/H = O.owner
	H.mutation_index--
```
Btw, is assigning organ owner on `uninstall()` intended?

Reply:
No, I think that was a copy and paste mistake

Reply:
```suggestion
/datum/component/modification/organ/uninstall(obj/item/organ/O, mob/living/user)
		..()
		if(istype(O, /obj/item/organ/internal/scaffold))
			var/obj/item/organ/internal/scaffold/S = O
			S.try_ruin()
	
		// If the organ has no owner or is still modded, do nothing
		if(!O.owner || LAZYLEN(O.item_upgrades))
			return
	
		O.owner.mutation_index--
```